### ISSUE TITLE
Software Elimination technique (e.g. distroless, removing unused packages, ...)

### ISSUE THREAD

**Issue by ioggstream** [d3fend/d3fend-ontology#82]
## I expect

SoftwareElimination technique. This extends the concept of DeadCodeElimination to platforms and container image.

## Note

imho when a software artifact is containerized, the deliverable is the whole container: this means that it could indeed fall into "dead code elimination" as long as the platform ops can't remove sw from the deliverable.

For platforms, removing unused packages/libs, it's indeed a different measure (e.g. DCE applied to another artifact).

cc: @netfl0

**Comment by BlueNTheFaceSSE** [comment]
Note ties to principle in NIST SP 800-160 Vol 1 Rev 1 - Least Functionality.

**Comment by netfl0** [comment]
Yes, this should go under platforms. lets do this next go-round.
